tests/integration/seccomp-notify-compat.bats

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	requires arch_x86_64
+
+	setup_seccompagent
+	setup_busybox
+}
+
+function teardown() {
+	teardown_seccompagent
+	teardown_bundle
+}
+
+# Support for seccomp notify requires Linux > 5.6, check that on older kernels
+# return an error.
+@test "runc run [seccomp] (SCMP_ACT_NOTIFY old kernel)" {
+	if [[ "$KERNEL_MAJOR" -gt 5 || ("$KERNEL_MAJOR" -eq 5 && "$KERNEL_MINOR" -ge 6) ]]; then
+		skip "requires kernel less than 5.6"
+	fi
```

Comment:
Maybe move this to `setup()` so it won't do any expensive setup if the kernel is not suitable?

Reply:
Sure, moved!